metagpt/actions/write_prd.py

```diff
@@ -123,7 +124,8 @@ async def _run_new_requirement(self, requirements, schema=CONFIG.prompt_schema)
         #     logger.info(rsp)
         project_name = CONFIG.project_name if CONFIG.project_name else ""
         context = CONTEXT_TEMPLATE.format(requirements=requirements, project_name=project_name)
-        node = await WRITE_PRD_NODE.fill(context=context, llm=self.llm)  # schema=schema
+        write_prd_node = WRITE_PRD_NODE if not project_name else WRITE_PRD_NODE_NO_NAME
```

Comment:
The original intention is to kick out the corresponding key when using, such as fill and exclude

Reply:
+ ActionNode.fill(..., exclude)